index.php

```diff
@@ -132,8 +132,43 @@ function getNews()
 $form = validForm(isset($_GET['fetch']) ? $_GET['fetch'] : '');
 if ($page == '') $page = '404';
 
-$lat = DEFAULT_LAT;
-$lon = DEFAULT_LON;
+$ip = $_SERVER['REMOTE_ADDR'];
+
+$url = 'http://api.lbs.yandex.net/geolocation';
+$apiKey = YLOCATOR_APIKEY;
```

Comment:
This should not be in `index.php`. I'm going to move it elsewhere, and it will be used as optional feature, not something called each time you open the map.